lib/game_session.rb

```diff
@@ -0,0 +1,79 @@
+# this class will make orchestrate interactions between all classes
+class GameSession
+
+  attr_accessor :player1, :player2, :board, :rules, :display, :translator, :current_player
+
+  def initialize(player1, player2, board, rules, display)
+    @player1 = Player.new("X")
+    @player2 = Player.new("O")
+    @board = Board.new(3, 3)
+    @rules = Rules.new(@board)
+    @display = Display.new(@board)
+    @translator = setup_translator
+    @current_player = @player1
+    # binding.pry
+    self.start_game
+
+  end
+
+  def start_game
+    p "Welcome to TicTacToe! Player 1 is 'X' and Player 2 is 'O'."
+    @display.display_board
+    # provide_move(@current_player) #comment for testing purposes
+  end
+
+  def provide_move(player)
+    print "Player #{player.mark}'s turn: "
```

Comment:
Is there a way we can remove the dependency to Ruby's print? Perhaps in the future, we want to repurpose the game to play over a browser instead of the console.